account_analytic_required/tests/test_account_analytic_required.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2014 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from datetime import datetime
+
+from odoo.tests import common
+from odoo import exceptions
+
+
+class TestAccountAnalyticRequired(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountAnalyticRequired, cls).setUpClass()
+        cls.account_obj = cls.env['account.account']
```

Comment:
@rven Add a global variable with the object and use it

Reply:
@nikul-serpentcs Change is done

Reply:
@nikul-serpentcs can you approve this PR?